lib/plotting_functions.py

```diff
@@ -733,8 +760,13 @@ def make_polar_plot(wks, case_nickname, base_nickname,
         norm1 = mpl.colors.Normalize(vmin=minval, vmax=maxval)
     #End if
 
-    if max(abs(levelsdiff)) > 10*absmaxdif:
+    if max(np.abs(levelsdiff)) > 10*absmaxdif:
         levelsdiff = np.linspace(-1*absmaxdif, absmaxdif, 12)
+    
+    #don't do this for the percent difference.
+    #if max(np.abs(levelspctdiff)) > 10*absmaxpct:
+    #    levelspctdiff = np.linspace(-1*absmaxpct, absmaxpct, 12)
+    
```

Comment:
Could you remove these commented out lines?

Reply:
Done